inc/class-cachify-hdd.php

```diff
@@ -99,7 +99,8 @@ public static function get_item() {
 	 */
 	public static function delete_item( $hash, $url ) {
 		self::_clear_dir(
-			self::_file_path( $url )
+			self::_file_path( $url ),
+			true
```

Comment:
When I added `$recursive = true`, I didn't really think about cached sub pages. My main concern was to delete the empty folder after invalidating the page cache.

These orphaned, empty folders aren't harmful, so I'll revert this change.

I think, it would be better to add an additional check, if the folder is empty. If so, the empty folder should be deleted by default - regardless of the `$recursive = false` option.

Reply:
For the empty folder I would simply remove the `if ($recursive )` condition before checking if the directory is empty. Deleting an empty directory in non-recursive is okay, no idea why there second condition exists.

Reply:
Yes, absolutely.
But I would rather do this in a separate change and merge this PR as is.

Reply:
Did so in #289 with some additional refactoring and unit tests. Didn't want to bloat your PR with all this stuff.